src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/GeneratedDocumentContainer.cs

```diff
@@ -139,12 +71,19 @@ public SourceTextContainer HtmlSourceTextContainer
         public bool TrySetOutput(
             DefaultDocumentSnapshot document,
             RazorCodeDocument codeDocument,
-            VersionStamp inputVersion,
-            VersionStamp outputCSharpVersion,
-            VersionStamp outputHtmlVersion)
+            VersionStamp inputVersion)
         {
             lock (_setOutputLock)
             {
+                if (_inputVersion.HasValue && _inputVersion.Value == inputVersion)
```

Comment:
Ah did you mean to leave the debug code here to check in?

Reply:
🤦‍♂️